app/controllers/api/connections_controller.rb

```diff
@@ -55,4 +64,104 @@ def request_match
 
     end
 
+    def update_match
+
+        if @current_user
```

Comment:
Like I said in PR #1, you can move this and lines 160-163 to a `before_action` block. Another advantage is that you only have to write this validation once and apply it to all methods instead of having these lines repeated across all of them